pymodbus/repl/client/main.py

```diff
@@ -46,9 +30,7 @@
 
 _logger = logging.getLogger()
 
-click.disable_unicode_literals_warning = True
-
-TITLE = f"""
+TITLE = rf"""
```

Comment:
You remove a unicode warning ???

Reply:
Yes, read the commit message.  https://github.com/pymodbus-dev/pymodbus/pull/1364/commits/00a58ca1bee36b3057c91e7b4fd40a0000570269